### PR TITLE
Additional Firely Terminal details

### PR DESCRIPTION
**Pull request by jafeltra**
Additional details about Firely Terminal, as suggested on Zulip [here](https://chat.fhir.org/#narrow/stream/215610-shorthand/topic/Firely.20Terminal.20'bake'.20command.20will.20become.20'inflate'/near/339823417).

### CODE REVIEW

File: content/docs/SUSHI/running/_index.md [hunk 1]
```diff
@@ -227,6 +227,12 @@ First, you must [install Firely Terminal](https://docs.fire.ly/projects/Firely-T
 You can see a list of the available Firely Terminal versions [here](https://www.nuget.org/packages/Firely.Terminal#versions-body-tab). Version 3.1.0 or higher is required to use the `inflate` command. Versions prior to 3.1.0 offer similar functionality via a `bake` command, but they are no longer recommended.
 {{% /alert %}}
 
+{{% alert title="Firely Terminal Note" color="info" %}}
+Inflating a single package, as described here with `fhir inflate --package  <packagename>`, is a free option in Firely Terminal. Using `fhir install <packagename>` will download, install, and automatically "inflate" the package. Using `fhir inflate` as a bulk option to inflate the current project and all its dependencies requires a Simplifier license.
```
Comment (cmoesel) [inline content/docs/SUSHI/running/_index.md:231]: Suggesting some minor tweaks to the wording:
```suggestion
Inflating a single package, as described here with `fhir inflate --package  <packagename>`, is a free option in Firely Terminal. Alternately, authors can use `fhir install <packagename>` to download, install, and automatically "inflate" the package. Using `fhir inflate` as a bulk option to inflate the current project and all its dependencies, however, requires a Simplifier license.
```

File: content/docs/SUSHI/running/_index.md [hunk 2]
```diff
@@ -227,6 +227,12 @@ First, you must [install Firely Terminal](https://docs.fire.ly/projects/Firely-T
...
+
+The [Package Management section](https://docs.fire.ly/projects/Firely-Terminal/Managing-Packages.html) of the Firely Terminal documentation provides additional details.
```
Comment (cmoesel) [inline content/docs/SUSHI/running/_index.md:233]: I don't recall, do we have a way to launch links in a separate tab/window? If so, can we / should we do that here?

**Review by cmoesel — COMMENTED** [review]
Thanks, @jafeltra! I've left a couple of minor suggestions for your consideration.

File: content/docs/SUSHI/running/_index.md (hunk 2 above)
Comment (jafeltra) [reply content/docs/SUSHI/running/_index.md:233]: I don't recall either, but that's a good idea so I'll check it out.

Comment (jafeltra) [reply content/docs/SUSHI/running/_index.md]: Done - [e4248a1](https://github.com/FSHSchool/site/pull/79/commits/e4248a1047acdf7148fdeae0c2c16f20137dd98c).

**Review by cmoesel — APPROVED** [review]
I dig it.

image

**Pull request merged**